docs/operator-manual/app-sync-using-impersonation.md

```diff
@@ -0,0 +1,114 @@
+
+## Introduction
+
+As of version 2.10, Argo CD supports syncing `Application` resources using the same service account used for its control plane operations. This feature enables users to decouple service account used for application sync from the service account used for control plane operations.
+
+Application syncs in Argo CD have the same privileges as the Argo CD control plane. As a consequence, in a multi-tenant setup, the Argo CD control plane privileges needs to match the tenant that needs the highest privileges. As an example, if an Argo CD instance has 10 Applications and only one of them requires admin privileges, then the Argo CD control plane must have admin privileges in order to be able to sync that one Application. Argo CD provides a multi-tenancy model to restrict what each Application can do using `AppProjects`, even though the control plane has higher privileges, however that creates a large attack surface since if Argo CD is compromised, attackers would have cluster-admin access to the cluster.
+
+Some manual steps will need to be performed by the Argo CD administrator in order to enable this feature. 
+
+!!! note
+    This feature is considered beta as of now. Some of the implementation details may change over the course of time until it is promoted to a stable status. We will be happy if early adopters use this feature and provide us with bug reports and feedback.
+
+### What is Impersonation
+
+Impersonation is a feature in Kubernetes and enabled in the `kubectl` CLI client, using which, a user can act as another user through impersonation headers. For example, an admin could use this feature to debug an authorization policy by temporarily impersonating another user and seeing if a request was denied.
+
+Impersonation requests first authenticate as the requesting user, then switch to the impersonated user info.
+
+```shell
+kubectl --as <user-to-impersonate> ...
+kubectl --as <user-to-impersonate> --as-group <group-to-impersonate> ...
+```
+## Prerequisites
+- In a multi team/multi tenant environment, an application team is typically granted access to a namespace to self-manage their Applications in a declarative way. 
+- The tenant namespace and the service account to be used for creating the resources in that namespace is created.
+- Create a Role to manage kubernetes resources in the tenant namespace
+- Create a RoleBinding to map the service account to the role created in the previous step.
+
+## Implementation details
+
+### Overview
+
+In order for an application to use a different service account for the application sync operation, the following steps needs to be performed:
+
+1. The impersonation feature flag should be enabled by setting the value of key `application.sync.impersonation.enabled` to `true` in the `argocd-cm` ConfigMap as below:
+```yaml
+data:
+  application.sync.impersonation.enabled: true
+```
```

Comment:
This is duplicated in `Enable application sync with impersonation feature` below. I feel we can just reference to that section from here.

Reply:
Removed the duplication and pointing to the relevant section.